src/BenchmarksApps/Kestrel/PlatformBenchmarks/BenchmarkApplication.Json.cs

```diff
@@ -7,6 +7,8 @@ namespace PlatformBenchmarks
 {
     public partial class BenchmarkApplication
     {
+        private readonly static uint _jsonPayloadSize = (uint)JsonSerializer.SerializeToUtf8Bytes(new JsonMessage { message = "Hello, World!" }, SerializerOptions).Length;
```

Comment:
@benaadams do you know if that would be an issue? I don't think so.

Reply:
No, similar approach in https://github.com/aspnet/Benchmarks/pull/1473 copied from other frameworks; feels a bit of a cheat, but doesn't look to be ruled out

Reply:
More specifically actrix is doing this; but also combining all the headers into one static chunk (which  #1473 does)

Reply:
Note I copied some of the changes from @adamsitnik's previous PR https://github.com/aspnet/Benchmarks/pull/1466 for #1473